Assets/Scripts/SlipTime/SlipTimeManager.cs

```diff
@@ -0,0 +1,68 @@
+﻿using UnityEngine;
+
+namespace SlipTime
+{
+    /// <summary>
+    /// Manages activating and deactivating SlipTime.
+    /// </summary>
+    public class SlipTimeManager : MonoBehaviour
+    {
+        /// <summary>
+        /// The percentage by which to scale time in SlipTime.
+        /// </summary>
+        public float slipTimeScalar = 0.25f;
+        
+        /// <summary>
+        /// The default duration of SlipTime in seconds.
+        /// </summary>
+        public float slipTimeDuration = 3f;
```

Comment:
The public properties allow us to change these values in the Unity editor without touching the underlying code. May be helpful for balancing.

Reply:
Yes, this is a good idea.